types/fabric/fabric-impl.d.ts

```diff
@@ -1710,6 +1710,47 @@ export class Group {
 	static fromObject(object: any, callback: (group: Group) => any): void;
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// ActiveSelection
+//////////////////////////////////////////////////////////////////////////////
+export interface ActiveSelection extends Object, ICollection<Group> {}
+export class ActiveSelection {
+	/**
+	 * Constructor
+	 * @param objects ActiveSelection objects
+	 * @param [options] Options object
+	 */
+	constructor(items?: any[], options?: IObjectOptions);
+
+	/**
+     * Change te activeSelection to a normal group,
+     * High level function that automatically adds it to canvas as
+     * active object. no events fired.
+     */
+	toGroup(): Group;
+
+	/**
+	 * Removes objects from a collection, then renders canvas (if `renderOnAddRemove` is not `false`)
+	 * @param object Zero or more fabric instances
+	 * @return thisArg
+	 * @chainable
+	 */
+	remove(...object: Object[]): Group;
+
+	/**
+	 * Returns string represenation of a group
+	 */
+	toString(): string;
+
+	/**
+	 * Returns {@link fabric.ActiveSelection} instance from an object representation
+	 * @memberOf fabric.ActiveSelection
+	 * @param object Object to create a group from
+	 * @param [callback] Callback to invoke when an ActiveSelection instance is created
+	 */
+	static fromObject(object: any, callback: (activeSelection: ActiveSelection) => any): void;
```

Comment:
If `fromObject` doesn't use the return value of callback, change it to `(activeSelection: ActiveSelection) => void`. Otherwise, is there a more precise type?

Reply:
As you say，changed it to (activeSelection: ActiveSelection) => void

Reply:
Do I need change the source once more?

Reply:
After the change of type, I have tested it locally. There is no problem.
Do I need to submit the code  once more?

Reply:
Please push your new changes! After that you should be all set 😀.